### ISSUE TITLE
Reddit Transcribers are commonly marked as bots, and YouAreDeaf marked as human

### ISSUE THREAD

**Issue by awlnx** [SM-Wistful/BotDetection-Algorithm#2]
Proposed solution:
1.
Implement a weight against submissions.
Active redditors generally have at least a few unique submissions.
Bots generally have  only 2 or 3 submissions dating back to their account creation.
2. weight against The parent comments
Bots generally  have predictable triggers, AreYouDeaf always responds to what

**Comment by SM-Wistful** [comment]
1 is already in effect, and I’ve been experimenting with 2. I’m probably going to take this weekend to retrain the model completely.

**Comment by DovidM** [comment]
Yeah, to add about the parent comments, the bot said /u/cryopodbot [isn’t a bot](https://www.reddit.com/r/TheCryopodToHell/comments/7sn7qj/comment/dt6c13n?st=JCY90NW2&sh=31d4747f) but crypodbot only responds to /u/klokinator posts, or posts in /r/klokinator (by copy/pasting klok’s posts or providing statistics)

**Comment by SM-Wistful** [comment]
I think this speaks to some larger issues with the way I trained the bot. When I initially made it, it was supposed to detect spammers and serve as a tool for moderators on Reddit. The idea was that if a bot only posted on one or two subreddits, it was probably a service/niche bot that wouldn't need to be banned. Obviously this is no longer the function of the bot, but many of the examples I gathered early on still reflect that influence. At some point I'm hoping to go through and add less biased examples as well.

With regard to 2, I have experimented with that, and one of the problems is that parent comments take a while to load. I was able to add a feature that detects if they reply very quickly, but I only used the last 10 comments. Doing it for more would be necessary to compare words, and I fear it might be too slow.
